static/scripts/dashboard/dashboard.html

```diff
@@ -1362,6 +1362,11 @@ <h4 class="panel-title">
                             <div class="row">
                               <div class="col-md-offset-2 col-md-2">
                                 isAvailable
+                                  <span>
+                                    <a title="This is available for the user to select form the dropdown in the subscription settings.">
+                                      <img src="https://shots.jotform.com/kade/Screenshots/blue_question_mark.png" height="13px" style="vertical-align: text-top !important"/>
```

Comment:
@Riturajcse Why is this getting an image from some random URL? Can we not just use font awesome?